roles/kafka_broker/tasks/health_check.yml

```diff
@@ -3,6 +3,8 @@
   shell: |
     {{ binary_base_path }}/bin/kafka-topics --bootstrap-server {{ hostvars[inventory_hostname]|confluent.platform.resolve_hostname }}:{{kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['port']}} \
       --describe --under-replicated-partitions --command-config {{kafka_broker.client_config_file}}
+  environment:
+    KAFKA_OPTS: "-Xlog:all=error -XX:+IgnoreUnrecognizedVMOptions"
```

Comment:
@mansisinha @utkarsh5474 
Wondering if this change should go in Kafka startup scripts ? Would non ansible users face this issue? If yes, lets add a JIRA against Kafka to track.

Reply:
yes, that sounds correct to me since this was a Java issue and not an ansible issue as per my understanding.